webapp/i18n/en.json

```diff
@@ -997,6 +997,7 @@
   "emoji_list.creator": "Creator",
   "emoji_list.delete": "Delete",
   "emoji_list.empty": "No Custom Emoji Found",
+  "emoji_list.help": "Custom emoji are available to everyone on your server and will show up in the emoji autocomplete menu. Other users may need to refresh the page before new emojis show up.",
```

Comment:
Super minor nit, maybe "show up" > "appear" in both instances above to make it easier to read for users for whom English isn't a first language?